assets/js/components/GoogleChart.js

```diff
@@ -126,13 +127,14 @@ export default function GoogleChart( props ) {
 		};
 	}, [
 		chart,
-		JSON.stringify( data ),
-		JSON.stringify( options ),
 		selectedStats,
 		singleStat,
+		chartID,
+		data,
+		options,
```

Comment:
same with this. `JSON.stringify` creates strings which are immutable and equal in JS if they have the same value, but that is not true with the object references I'm not passing in :thinking: 

The tests still pass but I'm worried in this file especially that I may have changed something (I'm happy to revert this and just disable eslint here)

Reply:
Note that this is a `useEffect`, the `useEffect` changes are what concern me!

Reply:
As we are re-running code (where as `useCallback` is just creating a new function that could be ran. so there are no side-effects)

Reply:
Is that a request @tofumatt ?  :smile:   I'm happy to disable eslint for any existing code

Reply:
@tofumatt  I reverted this file and just added an eslint ignore for this rule :+1: